typespec-extension/src/type-utils.ts

```diff
@@ -340,3 +356,23 @@ function getDecoratorScopedValue<T>(
   }
   return undefined;
 }
+
+export function isSdkIntKind(kind: string): boolean {
```

Comment:
TCGC exports two helpers: `isSdkIntKind` and `isSdkFloatKind`, will use theirs after they release.

Reply:
updated to use TCGC's helper function now.